### DISCUSSION TITLE
Default burndown chart for insights

### DISCUSSION THREAD

**Discussion by Bexolo** [community/community#51238] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

We are running sprints in GitHub Projects at the moment and cannot find a way to accurately report burndown progress. 
As a result, we would like to see a standard burndown chart included within the insights. 

![Screenshot 2023-03-28 at 15 13 23](https://user-images.githubusercontent.com/31006885/228266603-02765208-7fe8-4650-a59e-c88d137f04be.png)

This is the closest we can get to a burndown using insights, but it is really difficult to understand if we are on track based on the information in this graph. We want to see story point burndown progress based on when an issue is marked as 'done'.

A good example of the ZenHub burndown is here:

![Screenshot 2023-03-28 at 15 10 41](https://user-images.githubusercontent.com/31006885/228265753-38e09d1b-7d59-45a7-9064-e89d4987c61a.png)

**Comment by petewalker** [comment]
@Bexolo - how did you get time on the X-Axis to appear? I can't see that option in our project insights!

**Comment by andreaslech** [comment]
We faced the same challenge in GitHub: no Burndown charts and very limited flow metrics.
To solve this, I’ve built a lightweight add-on that makes it super simple: one click to get burndown and flow metrics directly from GitHub Projects.
The goal is to help teams validate process improvements and make retros more data-driven with metrics like Burndown, Cycle Time, Time in State.
Here’s an example of what it looks like:
<img width="649" height="360" alt="image" src="https://github.com/user-attachments/assets/fe7ce456-8edc-440a-ae57-80c84b023db8" />
If anyone is curious, I’ve shared more details here: [Flow2C](https://flow2c.com).
Would this type of visualization help your team? What other charts or metrics would you find useful?

**Comment by dtrezubSV** [comment]
+1. Insights charts in GitHub are almost useles and follow none of the standard charts one would expect from an agile management tool.

**Reply by andreaslech** [reply]
Same frustration here. I ended up building something for exactly this gap: cycle time, aging, sprint-over-sprint trends from GitHub Projects data. Still rough in places.

What's your team size? I'm curious how much the missing metrics actually hurt day-to-day.

**Comment by andreaslech** [comment]
Just tried generating a burndown chart for our sprint using GitHub Copilot, and it actually works really well!

<img width="400" height="300" alt="image" src="https://github.com/user-attachments/assets/fca17d31-7ebf-41f5-9021-a74b96c462ce" />

I ended up creating a small open-source repo with ready-to-use prompts for exactly this:
[github.com/Flow2CHQ/copilot-agile-metrics](https://github.com/Flow2CHQ/copilot-agile-metrics)

Burndown chart, sprint analysis, retro input, stakeholder update: out of the box, no extra tools needed. Works in VS Code and in the GitHub web UI.

Would love to hear what you think!
